docs/Configuration.md

```diff
@@ -292,8 +292,8 @@ Default value: `0`.
 ### age_reuse_previous_test
 
 Positive integer.
-How old (in seconds) a previous test of the same zone name and parameters must
-be before we start a new test.
+How old (in seconds) a previous test of the same zone name and parameters can be
+when it is reused instead of starting a new test.
```

Comment:
Not sure if `can` conveys the right meaning. From my understanding if the time limit is reached, then a new test is run, otherwise the old test's results are displayed. And the usage of _can_ here make it sounds like there is a possibility to run the test even if the time limit is not reached.

I understand that there might be something unclear with the current sentence. 
If you feel we need to rewrite this sentence, I tried to formulate it another way (but I'm still not satisfied)
-> Time (in seconds) to wait before running a new test instead of reusing the results of a previous test of the same zone name and parameter.

Reply:
When I tried to look for a real life analogy to what we're trying to express and the closes I came was "shelf life". I'll come up with a new wording based on that.